src/main/java/uk/gov/hmcts/reform/refunds/controllers/RefundController.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.refunds.controllers;
+
+import io.swagger.annotations.Api;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.refunds.model.RefundReason;
+import uk.gov.hmcts.reform.refunds.repository.RefundReasonsRepository;
+
+import java.util.List;
+
+
+/**
+ * Refund controller for backend rest api operations
+ */
+@RestController
+@Api(tags = {"Refund Journey group"})
+public class RefundController {
+
+    @Autowired
+    RefundReasonsRepository refundsReasonsRepository;
```

Comment:
you should never calling repository layer from controller. Please create a service layer and call service layer from controller

Reply:
Done the changes as per the review comments, added the service layer